runtime/tests/runtime_integration_tests/test_boot.rs

```diff
@@ -161,11 +164,19 @@ fn test_stress_update() {
 
 #[test]
 fn test_boot_tci_data() {
+    let fw_id = if cfg!(all(feature = "fpga_realtime", feature = "fpga_subsystem")) {
```

Comment:
Is this supposed to be `any`?
```suggestion
    let fw_id = if cfg!(any(feature = "fpga_realtime", feature = "fpga_subsystem")) {
```

Reply:
https://github.com/chipsalliance/caliptra-sw/pull/2637

Reply:
Thank you!